packages/frontend-api/src/Model/Mutation/BaseTokenMutation.php

```diff
@@ -31,14 +31,14 @@ protected function runCheckUserIsLogged(): FrontendApiUser
         $token = $this->tokenStorage->getToken();
 
         if ($token === null) {
-            throw new UserError('Token is not valid.');
+            throw new InvalidTokenUserError('Token is not valid.');
```

Comment:
Tokens are a little confusing to me right now. We have also `TokenUserMessageException` and its descendants that also prints an error to the user. Wouldn't be nice to unify it somehow? 

For example `TokenFacade::getTokenByString()` also returns "Token is not valid" message, but without code.

Reply:
good point, what about this? 35feb68ce